rust-toolchain.toml

```diff
@@ -0,0 +1,5 @@
+[toolchain]
+channel = "1.87.0"
```

Comment:
I feel like we should be using the latest stable, which is 1.88.

Reply:
Bumped in [ec41565](https://github.com/BitcoinL2-Labs/sPoX/pull/2/commits/ec415659231089fd52c2f75b57515ba27c6dbe22)